### PR TITLE
Adds unit tests for Page component

### PR DESCRIPTION
**Pull request by substantial-kst**
Additionally makes minor refactors to:
* Episode interface
* Header / Browse styling
* Jest coverage requirements

### CODE REVIEW

File: src/pages/Page/Page.test.tsx [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+                });
+                it(`/ProgramId/s04e99 -> Fake Episode`, () => {
+                    expect(sut(DetailPageRouteProps))
+                        .toStrictEqual(fakeEpisode);
+                });
+            });
+        });
+    });
+
+    // describe('rendered component', () => {
```
Comment (substantial-kst) [inline src/pages/Page/Page.test.tsx]: Any rendered test I can come up with for `Page` seems more appropriate for the child components (`LandingPage`, `Browse`, `Search`, or `Detail`).

Presently just testing the internals yields pretty good coverage:
```
--------------|----------|----------|----------|----------|-------------------|
File          |  % Stmts | % Branch |  % Funcs |  % Lines | Uncovered Line #s |
--------------|----------|----------|----------|----------|-------------------|
All files     |    46.75 |       65 |    19.05 |    48.65 |                   |
 components   |    36.36 |        0 |        0 |    36.36 |                   |
  Header.tsx  |    36.36 |        0 |        0 |    36.36 |... 34,35,43,48,49 |
 pages/Browse |    11.11 |        0 |        0 |     12.5 |                   |
  Browse.tsx  |    11.11 |        0 |        0 |     12.5 |... 57,58,62,63,65 |
 pages/Detail |     6.25 |        0 |        0 |     6.67 |                   |
  Detail.tsx  |     6.25 |        0 |        0 |     6.67 |... 42,43,45,53,61 |
 pages/Page   |    90.63 |    92.86 |       80 |    90.63 |                   |
  Page.tsx    |    90.63 |    92.86 |       80 |    90.63 |          64,66,67 |
--------------|----------|----------|----------|----------|-------------------|
```

File: src/pages/Detail/Detail.tsx [hunk 2]
```diff
@@ -6,13 +6,20 @@ import TextSummary from '../../components/TextSummary';
 import BroadcastDate from '../../components/BroadcastDate';
 import GuestStars from '../../components/GuestStars/GuestStars';
 import LoadingIndicator from '../../components/LoadingIndicator';
+import styled from "@emotion/styled";
 
 interface Props {
-  episode: Episode;
+  episode: Episode|null;
```
Comment (jlee0831) [inline src/pages/Detail/Detail.tsx]: I think you can do: `episode?: Episode`, but I can't remember if that matches to `null` or to `undefined`.

Comment (substantial-kst) [reply src/pages/Detail/Detail.tsx]: Seems like `undefined`; the `|null` got added to get around that, and the `null` value's used in this case to indicate an explicit 'Parent does not have an episode to give you, <Details /> child - you will need to get it yourself.'

Comment (substantial-kst) [reply src/pages/Detail/Detail.tsx]: No matter how the rest of the day goes, today I learned that it's possible to accidentally create a collapsable text component in GitHub comments.

**Pull request merged**